.circleci/config.yml

```diff
@@ -14,10 +14,7 @@ references:
 executors:
     node-executor:
         docker:
-            - image: circleci/node:14.15.4-browsers
-    docker-terraform:
-        docker:
-            - image: "hashicorp/terraform:light"
+            - image: circleci/python:3.10-rc-buster-node-browsers-legacy
```

Comment:
Is there a reason why we are using this specific python image version?

Reply:
It wasn't able to install AWS otherwise. (Unless I was doing something wrong). It was looking for pip? Once I changed it to this image it worked.